docs/entanglements.md

```diff
@@ -83,6 +83,10 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: link-test-nats-nats
+  annotations:
+    quarks.cloudfoundry.org/update-referenced-owner: "true"
```

Comment:
I don't think the name of the label makes sense - "update referenced owner".
We're not updating an "owner". Please rename to `quarks.cloudfoundry.org/restart-on-update`.

Reply:
Done.